clr-installer/main.go

```diff
@@ -43,13 +43,6 @@ var (
 )
 
 func fatal(err error) {
-	if lock != "" {
```

Comment:
One of the deferred functions will unlock the lock. Also, the global lock value is never set, so this condition will never be true (The lock value is set with a local scope, so it does not set the global scope).